sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoderSuite.scala

```diff
@@ -248,7 +248,7 @@ class ExpressionEncoderSuite extends CodegenInterpretedPlanTest with AnalysisTes
       useFallback = true)
   }
 
-  object OuterLevelWithVeryVeryVeryLongClassName1 {
+  /* object OuterLevelWithVeryVeryVeryLongClassName1 {
```

Comment:
Why did you comment it?

Reply:
Building on my machine was failing because of this. Pushed it by accident, resolved now.